cassiopeia/data/utilities.py

```diff
@@ -1,7 +1,9 @@
 """
 General utilities for the datasets encountered in Cassiopeia.
 """
-from typing import Callable, Dict, List, Optional, Tuple
+import collections
+import warnings
+from typing import Callable, Dict, List, Optional, Tuple, Union
```

Comment:
We introduce imports alphabetically - can you move `typing` above `warnings`?

Reply:
Ha. Seems like I forgot my abc's :sweat_smile: